template.yaml

```diff
@@ -0,0 +1,18 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  GetMeASeatAt:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: "<YOUR HANDLER>"
```

Comment:
Do we need to fill a handler?

Reply:
I'm trying to make it generic, since it will be used by other developers as a part of AWS Labs. However, those values would need to be filled in to deploy the package.

Reply:
Make sense.
Ship it ;)

Reply:
E.g. <bucket> in https://github.com/awslabs/serverless-application-model/blob/master/examples/2016-10-31/alexa_skill/template.yaml

We can revisit the template later, when we are close to publishing and figured out the name of the handler, jar, etc.